tools/gui/qtkontakt.py

```diff
@@ -325,6 +330,7 @@ def __lade_alle_werte(self):
                        "Datei, indem Sie auf Speichern klicken.")
 
         except ValidationError as exc:
+            self.__reset_vermittlungscodes
```

Comment:
Wird nicht aufgerufen :)
```suggestion
            self.__reset_vermittlungscodes()
```